strawberry_django/fields/types.py

```diff
@@ -102,6 +103,31 @@ class ManyToManyInput:
         }
     )
 
+try:
+    from django.contrib.gis.db import models as geos_fields
+
+    Point = strawberry.scalar(
+        NewType("Point", Tuple[float, ...]),
+        serialize=lambda v: v,
+        parse_value=lambda v: v,
+    )
+
+    field_type_map.update(
+        {
+            geos_fields.PointField: Point,
+            geos_fields.LineStringField: List[Point],
+            geos_fields.PolygonField: List[Point],
+            geos_fields.MultiPointField: List[Point],
+            geos_fields.MultiLineStringField: List[List[Point]],
+            geos_fields.MultiPolygonField: List[List[Point]],
+        }
+    )
+except django.core.exceptions.ImproperlyConfigured:
+    # in gdal is not available
+    # skip
```

Comment:
I think you mean `if`, not `in`. Also, one line for this comment should be enough, like `# If gdal is not available, skip.`